### DISCUSSION TITLE
option that allows you to change the reminder confirmation button to a long press, swipe, double tap, etc.

### DISCUSSION THREAD

**Discussion by fzrGit** [Bitfire-Development/snooze-it-feedback#158] [Ideas]
It's very convenient to use. However, you may accidentally tap the screen when taking your smartphone out of your pocket. Could you please implement an option that allows you to change the reminder confirmation button to a long press, swipe, double tap, etc. Thank you for your confirmation.

**Comment by gmm6797** [comment]
Why not turn on settings|reminders|all groups|hide reminder when device is locked

**Reply by fzrGit** [reply]
Thank you for letting me know it was the feature I was looking for

**Reply by gmm6797** [reply]
You are welcome, glad to help

**Reply by Bitfire-Development** [reply]
Thanks @gmm6797 for helping me out in answering this question! I really appreciate it :)

**Reply by gmm6797** [reply]
Always happy to help others, and I knew "an answer", maybe not the best one :)

**Comment by Bitfire-Development** [comment]
I'm glad to see you have fixed your issue using the "hide reminder when device is locked" feature. I do feel however like this might not be the best solution for all users, so I will keep this idea open for voting.

I do think this would be a good addition to the app, but am focusing my limited time on other bugs/features for ow. If this idea gets a lot of votes I will put it higher on my to-do list. (hopefully I can bring out a new update soon, I know it's been quite a while...).
